frontend/packages/dev-console/src/components/pipelines/resource-types/triggers.ts

```diff
@@ -27,9 +27,16 @@ export type TriggerTemplateKind = K8sResourceCommon & {
 };
 
 export type EventListenerKindBindingReference = {
-  // TriggerBinding / ClusterTriggerBinding name reference
-  name: string;
-  kind?: string;
+  // TriggerBinding / ClusterTriggerBinding reference
+  kind: string;
+  // Ref is used since Tekton Triggers 0.5 (part of OpenShift Pipeline Operator 1.1)
+  ref: string;
+  // We also support older operators, so need to show & save the old field as well.
+  // https://github.com/tektoncd/triggers/pull/603/files
+  // https://github.com/tektoncd/triggers/releases/tag/v0.5.0 and
+  // https://github.com/tektoncd/triggers/releases/tag/v0.6.0
+  /** @deprecated use ref instead */
+  name?: string;
```

Comment:
Wondering if we might be able to use `keyof` here, or would it become too verbose? Especially since `name` is deprecated.

Reply:
Don't know what the solution looks like then. But the usage in triggers.ts works fine with name and at least VScode shows these deprecated hint. But I'm open to see and try an example here. /me open the TS keyof definition :)